src/style.css

```diff
@@ -0,0 +1,57 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+html {
+  font-weight: 300;
+}
+
+.conteiner {
+  display: block;
+  margin: 20px 0 0;
+}
+
+.searchInp {
+  font-family: Avenir, sans-serif;
+  box-sizing: border-box;
+  width: 100%;
+  /*margin: 20px 0 0;*/
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.searchInp::placeholder {
+  font-family: Avenir, sans-serif;
+}
+
+.searchInp:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.searchInp:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.bigSearchInp {
+  font-size: 16px;
+  line-height: 21.86px;
+  padding-left: 62px;
+  height: 70px;
+  background: #fff url(./images/Search.svg) no-repeat;
```

Comment:
🔥 
but its the same value for both inputs, move it to `.searchInp`

Reply:
its the same parameters, but not value